source/RN/index.rst

```diff
@@ -0,0 +1,10 @@
+.. _rel_notes:
+
+==================================================
+Kubernetes for Percona Server for MongoDB |version| Release Notes
+==================================================
+
+.. toctree::
+   :maxdepth: 1
+
+   1.0.0
```

Comment:
what about 0.3.0, 0.2.1, 0.2.0, 0.1.0 release notes?

Reply:
Peter S said that we start with RN 1.0.0